docs/pages/router/installation.mdx

```diff
@@ -14,9 +14,9 @@ Find the steps below to create a new project with Expo Router library or add it
 
 <Step label="1">
 
-We recommend creating a new Expo app using `create-expo-app`. This will create a minimal project with the Expo Router library already installed. To create a project, run the command:
+We recommend creating a new Expo app using `create-expo-app`. Starting in SDK 51, the default template includes a minimal project with the Expo Router library already installed. To create a project, run the command:
```

Comment:
```suggestion
We recommend creating a new Expo app using `create-expo-app`. To create a project, run the command:
```

Since this is a "Quickstart" step, there is no need to explain that the template will create a new project with Expo Router already installed. Any command we offer here, it is expected to create a new project with Expo Router library installed.

Reply:
Instead we should change the command in the "Manual installation" to use a blank project. 
![CleanShot 2024-05-11 at 02 59 11@2x](https://github.com/expo/expo/assets/10234615/47cb69d7-b855-42c2-95d9-e4cd4f2ee7e9)

The `npx create-expo-app` will also use the new default template.

But this brings up the question, if I'm creating a new project, why would I create a blank project and then install Expo Router following the manual steps. Ideally, since Expo Router library is now a part of the default template, under "Manual installation", we should consider removing the step of "Creating a new project" and instead say, the steps under Manual Installation assume the user already has a project created with Expo (but doesn't have Expo Router installed).

The current context of this page used to fit when Expo Router was only available with "tabs" template.